bower.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "marketplace-frontend": "7.0.1",
```

Comment:
Does `bower install` work for this without issues? I think this is what caused the issues in marketplace-elements.